src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,98 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private int size = 0;
+    private T[] objects;
```

Comment:
```suggestion
    private T[] objects;
```
I'd recommend renaming to 'elements' or 'values'